app/src/controllers/fetch/reducer.js

```
@@ -23,35 +23,37 @@ const computeInitialState = (options) => {
+    state = computeInitialState(options),
+    { type = '', payload = {}, meta = {}, concat = false },
+  ) => {
+    if (meta?.namespace !== namespace) {
+      return state;
     }
-    case CONCAT_FETCH_SUCCESS: {
-      const data = contentPath ? payload[contentPath] : payload;
+    const contentPath = options.contentPath || DEFAULT_OPTIONS.contentPath;
+    switch (type) {
+      case FETCH_SUCCESS: {
+        if (options.getFirst && payload[contentPath] instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)